src/System.Drawing.Primitives/src/System/Drawing/Point.cs

```diff
@@ -0,0 +1,280 @@
+        ///       of the <see cref='System.Drawing.Point.X'/> or <see cref='System.Drawing.Point.Y'/> properties of the two
+        ///    <see cref='System.Drawing.Point'/> 
+        ///    objects are unequal.
+        /// </para>
+        /// </devdoc>
+        public static bool operator !=(Point left, Point right)
+        {
+            return !(left == right);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.Point'/> by a given <see cref='System.Drawing.Size'/> .
+        ///    </para>
+        /// </devdoc>        
+        public static Point Add(Point pt, Size sz)
+        {
+            return new Point(pt.X + sz.Width, pt.Y + sz.Height);
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Translates a <see cref='System.Drawing.Point'/> by the negative of a given <see cref='System.Drawing.Size'/> .
+        ///    </para>
+        /// </devdoc>        
+        public static Point Subtract(Point pt, Size sz)
+        {
+            return new Point(pt.X - sz.Width, pt.Y - sz.Height);
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a ceiling operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Ceiling(PointF value)
+        {
+            return new Point((int)Math.Ceiling(value.X), (int)Math.Ceiling(value.Y));
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a truncate operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Truncate(PointF value)
+        {
+            return new Point((int)value.X, (int)value.Y);
+        }
+
+        /// <devdoc>
+        ///   Converts a PointF to a Point by performing a round operation on
+        ///   all the coordinates.
+        /// </devdoc>
+        public static Point Round(PointF value)
+        {
+            return new Point((int)Math.Round(value.X), (int)Math.Round(value.Y));
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Specifies whether this <see cref='System.Drawing.Point'/> contains
+        ///       the same coordinates as the specified <see cref='System.Object'/>.
+        ///    </para>
+        /// </devdoc>
+        public override bool Equals(object obj)
+        {
+            if (!(obj is Point)) return false;
+            Point comp = (Point)obj;
+            // Note value types can't have derived classes, so we don't need 
+            // to check the types of the objects here.
+            return comp.X == X && comp.Y == Y;
+        }
+
+        /// <devdoc>
+        ///    <para>
+        ///       Returns a hash code.
+        ///    </para>
+        /// </devdoc>
+        public override int GetHashCode()
+        {
+            return unchecked(_x ^ _y);
```

Comment:
Is the library being compiled with checking explicitly enabled?  Assuming not (since the default is unchecked), this and other uses of ```unchecked``` aren't necessary.  They don't hurt, but we also don't appear to be using it consistently across the project, which makes it confusing when it is used.